File: frontend-angular/src/app/models/habitacion-dto.ts
export enum TipoCama {
  INDIVIDUAL = 'INDIVIDUAL',
  DOBLE = 'DOBLE',
  QUEEN = 'QUEEN',
  KING = 'KING'
}

export interface HabitacionDto {
  codigo: number;
  numeroCamas: number;
  tipoCama: TipoCama;
  tieneBano: boolean;
}

File: src/main/java/com/edu/uniquindio/ruralstay/dto/HabitacionDTO.java
package com.edu.uniquindio.ruralstay.dto;

import com.edu.uniquindio.ruralstay.entity.enums.TipoCama;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class HabitacionDTO {
    private Long codigo;
    private Integer numeroCamas;
    private TipoCama tipoCama;
    private Boolean tieneBano;
}
